sonar/modules/documents/serializers/__init__.py

```diff
@@ -73,6 +75,28 @@ def post_process_serialize_search(self, results, pid_fetcher):
             if organisation:
                 org_term['name'] = organisation['name']
 
+        # Process facets for custom fields
```

Comment:
I think this can be done in the query factory. If we do not need facets in some view, why to compute it?